### PR TITLE
Added `Using new features` wiki page

### PR DESCRIPTION
**Pull request by freya022**
This PR adds the "Using new features" wiki page, which uses JitPack to try out PR branches

### CODE REVIEW

File: mkdocs.yml [hunk 1]
```diff
@@ -76,6 +76,7 @@ nav:
     - Gateway Intents and Member Cache Policy: using-jda/gateway-intents-and-member-cache-policy.md
     - Webhooks: https://github.com/MinnDevelopment/discord-webhooks
     - Managers and Moderation: using-jda/separation-of-concerns.md
+    - Using new features: using-jda/using-new-features.md
```
Comment (oliver276) [inline mkdocs.yml:79]:
```suggestion
    - Using New Features: using-jda/using-new-features.md
```

File: docs/using-jda/using-new-features.md [hunk 2]
```diff
@@ -0,0 +1,78 @@
+# Using new features
```
Comment (oliver276) [inline docs/using-jda/using-new-features.md:1]:
```suggestion
# Using New Features
```

File: docs/using-jda/using-new-features.md [hunk 3]
```diff
@@ -0,0 +1,78 @@
...
+
+When contributors are making new features, for example, introducing modals, 
+you might be interested in trying these features before they get released and/or test them.
```
Comment (oliver276) [inline docs/using-jda/using-new-features.md:4]:
```suggestion
When contributors are making new features, like introducing modals, 
you might be interested in trying out or testing these features before they get released.
```

File: docs/using-jda/using-new-features.md [hunk 4]
```diff
@@ -0,0 +1,78 @@
...
+
+## Using [JitPack](https://jitpack.io)
+
+Using JitPack, you can get a build of any fork of JDA, on any branch, on any commit.
+Of course, it is highly recommended that you use a build tool such as [Gradle](https://gradle.org) or [Maven](https://maven.apache.org)
```
Comment (oliver276) [inline docs/using-jda/using-new-features.md:9]:
```suggestion
It is still highly recommended that you use a build tool such as [Gradle](https://gradle.org) or [Maven](https://maven.apache.org)
```

File: docs/using-jda/using-new-features.md [hunk 5]
```diff
@@ -0,0 +1,78 @@
...
+
+### Getting the necessary info
```
Comment (oliver276) [inline docs/using-jda/using-new-features.md:11]: Please capitalise these headings

File: docs/using-jda/using-new-features.md [hunk 6]
```diff
@@ -0,0 +1,78 @@
...
+
+You have to then click the arrow, and select the branch you want, here we'll choose the `feature/modals` branch, you should have something like that
+
+![img.png](../assets/images/jitpack_selected_branch.png){width="480"}
+
+You can then pick the version that's second from top to bottom, so here it would be `ef20a4069e`, this is the **commit hash**
+
+### Putting it together
+
+So far we got:
```
Comment (oliver276) [inline docs/using-jda/using-new-features.md:40]:
```suggestion
So far we have:
```

**Review by oliver276 — CHANGES_REQUESTED** [review]

**Review by oliver276 — APPROVED** [review]

File: docs/using-jda/using-new-features.md [hunk 7]
```diff
@@ -0,0 +1,71 @@
...
+Using JitPack, you can get a build of any fork of JDA, on any branch, on any commit.
+It is still highly recommended that you use a build tool such as [Gradle](https://gradle.org) or [Maven](https://maven.apache.org)
+
+### Getting the Necessary Info
+
+Let's take the [PR for the Discord modals](https://github.com/DV8FromTheWorld/JDA/pull/2024) as an example
+
+![img.png](../assets/images/github_pr.png){width="720"}
+
+Here the important part is `from Xirado:feature/modals`, you need to extract the PR author, which here is `xirado`
```
Comment (oliver276) [inline docs/using-jda/using-new-features.md:17]: This sentence doesn't seem to make much sense.  I think it is fine to just say something like "You need the author's name, which here is \`Xirado\`."

File: docs/using-jda/using-new-features.md [hunk 8]
```diff
@@ -0,0 +1,71 @@
...
+
+### Getting the Latest Version
+
+You then want to go in the `Commits` tab
```
Comment (oliver276) [inline docs/using-jda/using-new-features.md:21]:
```suggestion
Next, head to the `Commits` tab.
```

File: docs/using-jda/using-new-features.md [hunk 9]
```diff
@@ -0,0 +1,71 @@
...
+
+![img.png](../assets/images/github_pr_commits_tab.png){width="720"}
+
+You can now scroll down to the bottom, as to find the latest **commit hash**, it would look like this:
```
Comment (oliver276) [inline docs/using-jda/using-new-features.md:25]:
```suggestion
You can now scroll down to the bottom, as to find the latest **commit hash**.  It should look like this:
```

File: docs/using-jda/using-new-features.md [hunk 10]
```diff
@@ -0,0 +1,71 @@
...
+### Putting It Together
+
+So far we have:
+
+* PR author name: `xirado`
+* Version (7 char commit hash): `36eea2a`
+
+You can now add the dependency to your favorite build tool, but before that you will have to remove your current JDA dependency
+
+=== "Maven"
```
Comment (oliver276) [inline docs/using-jda/using-new-features.md:40]: Elsewhere in the wiki, Gradle has come before Maven.  This is probably something we should standardise.

**Review by oliver276 — CHANGES_REQUESTED** [review]
There are a few missing full stops/periods here.

It also feels a bit odd to continue the same example of Xirado's PR without showing what that example would look like at the end.

**Review by oliver276 — APPROVED** [review]

File: docs/introduction/faq.md [hunk 11]
```diff
@@ -192,49 +192,7 @@ Didn't find an answer? Try asking in [our Discord server](https://discord.gg/0hM
...
-        </repository>
-        ```
-        ```xml
-        <dependency>
-            <groupId>com.github.DV8FromTheWorld</groupId>
-            <artifactId>JDA</artifactId>
-            <version>VERSION</version>
-        </dependency>
-        ```
+    You can use JitPack to provide other branches of JDA as a dependency.  We have a tutorial on how to use it on [this wiki page](/using-jda/using-new-features)
```
Comment (Aikovdp) [inline docs/introduction/faq.md:195]:
```suggestion
    You can use JitPack to provide other branches of JDA as a dependency.  We have a tutorial on how to use it on [this wiki page](/using-jda/using-new-features).
```

File: docs/using-jda/using-new-features.md [hunk 12]
```diff
@@ -0,0 +1,108 @@
...
+
+Next, head to the `Commits` tab.
+
+![img.png](../assets/images/github_pr_commits_tab.png){width="720"}
+
+You can now scroll down to the bottom, as to find the latest **commit hash**.  It should look like this:
+
+![img.png](../assets/images/github_pr_latest_commit.png){width="720"}
+
+Here you can see the 7 first chars of the latest commit are `36eea2a`.
```
Comment (Aikovdp) [inline docs/using-jda/using-new-features.md:29]:
```suggestion
Here you can see the 7 first characters of the latest commit are `36eea2a`.
```

File: docs/using-jda/using-new-features.md [hunk 13]
```diff
@@ -0,0 +1,108 @@
...
+
+### Putting It Together
+
+So far we have:
+
+* PR author name: `xirado`
+* Version (7 char commit hash): `36eea2a`
+
+You can now add the dependency to your favorite build tool, but before that, you will have to remove your current JDA dependency.
```
Comment (Aikovdp) [inline docs/using-jda/using-new-features.md:38]:
```suggestion
You can now add the dependency to your project using your favorite build tool, but don't forget to remove your current JDA dependency first.
```

File: docs/using-jda/using-new-features.md [hunk 14]
```diff
@@ -0,0 +1,108 @@
...
+![img.png](../assets/images/github_pr_latest_commit.png){width="720"}
+
+Here you can see the 7 first chars of the latest commit are `36eea2a`.
+
+### Putting It Together
+
+So far we have:
+
+* PR author name: `xirado`
+* Version (7 char commit hash): `36eea2a`
```
Comment (Aikovdp) [inline docs/using-jda/using-new-features.md:36]:
```suggestion
* Version (7 character commit hash): `36eea2a`
```

File: mkdocs.yml [hunk 15]
```diff
@@ -44,6 +44,8 @@ markdown_extensions:
   - pymdownx.highlight
   - footnotes
   - def_list
+  - attr_list
+  - md_in_html
```
Comment (Aikovdp) [inline mkdocs.yml:48]: I don't think the `md_in_html` extension is being used here, only the `attr_list` one is necessary for those images. 
```suggestion
```

**Review by Aikovdp — COMMENTED** [review]
Looks good to me! Just a few minor suggestions, you can do with those what you want.

**Review by Aikovdp — APPROVED** [review]

**Pull request merged**